pkg/events/logger_handler/logger_handler.go

```diff
@@ -0,0 +1,101 @@
+package logger_handler
+
+import (
+	k8sV1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/events"
+)
+
+type LoggerHandler struct {
+	events.EventHandlerBase
+}
+
+// GetName returns the name of the event handler
```

Comment:
We don't need to duplicate the interface method docs here also it can become out-of-date. Same below.

Reply:
:+1: